### PR TITLE
Fix bug with github url on template pages

### PR DESCRIPTION
**Pull request by victoriabernard92**

### CODE REVIEW

**Comment by gabbifish** [comment]
Can we deploy this to staging first to make sure it works? :D

File: layouts/partials/template-page.html [hunk 1]
```diff
@@ -82,7 +82,7 @@ <h2>
        {{ $repo_name := $repo_json.full_name}}
        {{ $repo_date := $repo_json.updated_at}}
        <div class="github">
-        <a href="{{$repo_url}}">
+        <a href="{{.repository_url}}">
```
Comment (signalnerve) [inline layouts/partials/template-page.html]: just to make sure i understand, the `$repo_url` is a substitution on `repository_url` that is set to `api.github.com/repos`, right? makes sense to use `repository_url` for a link instead, just want to make sure i get what's going on

Comment (victoriabernard92) [reply layouts/partials/template-page.html]: correct

`repo_url` is modified off of `repository_url` which comes from the registry. I should improve variable naming

Comment (victoriabernard92) [reply layouts/partials/template-page.html]: All fixed :)

**Comment by victoriabernard92** [comment]
> Can we deploy this to staging first to make sure it works? :D

No need this is just a hugo thing doesn't impact the worker

**Pull request merged**